cleaning_services/__manifest__.py

```diff
@@ -0,0 +1,69 @@
+{
+    'name': 'Cleaning Services',
+    'version': '1.0',
+    'category': 'Services',
+    'description': '',
+    'author': 'Odoo S.A.',
+    'depends': [
+        'account_online_synchronization',
+        'crm_sale_subscription',
+        'hr_skills',
```

Comment:
Maybe check with the PO if hr is needed or not (and remove all hr related records if not)
```suggestion
        'hr',
```